src/neptune/handler.py

```diff
@@ -557,7 +557,14 @@ def fetch_last(self):
         """
         return self._pass_call_to_attr(function_name="fetch_last")
 
-    def fetch_values(self, *, include_timestamp: Optional[bool] = True, progress_bar: Optional[ProgressBarType] = None):
+    def fetch_values(
+        self,
+        *,
+        include_timestamp: Optional[bool] = True,
+        progress_bar: Optional[ProgressBarType] = None,
+        offset: Optional[int] = None,
```

Comment:
There were comments on Jira to use `begin` and `end` instead. Like Python does for lists.

Reply:
oh it's in the Jira comment. I missed that 😨